circle.yml

```diff
@@ -12,7 +12,6 @@ dependencies:
     - security find-identity -p codesigning
     - instruments -s devices
     - xcodebuild -showsdks
-    - gem install fastlane -NVv 2.37.0
```

Comment:
why do we need to delete this line?

Reply:
Circle fastlane now by default and the version that they provide is good for what we need 👍